ppml/trusted-big-data-ml/python/docker-gramine/bigdl-gramine/Dockerfile

```diff
@@ -6,14 +6,15 @@ FROM $BASE_IMAGE_NAME:$BASE_IMAGE_TAG as temp
 
 ARG SGX_MEM_SIZE
 ARG SGX_LOG_LEVEL
+ARG ATTESTATION
```

Comment:
Attestation change to ENABLE_DCAP_ATTESTATION

Reply:
fixed.